pkg/pubsub/adapter/converters/auditlog_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package converters
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	cepubsub "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub"
+	pubsubcontext "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/pubsub/context"
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	auditpb "google.golang.org/genproto/googleapis/cloud/audit"
+	logpb "google.golang.org/genproto/googleapis/logging/v2"
+)
+
+var (
+	tctx = pubsubcontext.TransportContext{
+		ID:      "test-id",
+		Project: "test-project",
+		Topic:   "test-topic",
+	}
+)
+
+func TestConvertAuditLog(t *testing.T) {
```

Comment:
Can we also test other cases like non-AuditLog log entry?

Reply:
Done.